vpr/src/pack/attraction_groups.cpp

```diff
@@ -0,0 +1,38 @@
+#include "attraction_groups.h"
+
+AttractGroupId AttractionInfo::get_atom_attraction_group(AtomBlockId atom_id) {
+    return atom_attraction_group[atom_id];
+}
+
+AttractionGroup AttractionInfo::get_attraction_group_info(AttractGroupId group_id) {
+    return attraction_groups[group_id];
+}
+
+void AttractionInfo::set_atom_attraction_group(AtomBlockId atom_id, AttractGroupId group_id) {
```

Comment:
see if you should also add the atom in the other data structure (the vector of atoms that are part of an attraction group)

Reply:
may not need these accessors, can also just give the callers direct access to the data structures